cap32.cfg

```diff
@@ -50,7 +50,7 @@ joystick_menu_button=9
 joystick_vkeyboard_button=10
 # resources_path
 #   path to resources (menu images...)
-resources_path=/usr/local/share/caprice32/resources
+resources_path=
```

Comment:
i wonder if i could create a CAP32_APPPATH or DATAPATH pointing on default RESSOURCE i could use in the cfg. Btw i'd like to merge this _as it is_ because i mess with git (forget to create branch per feature, and need to write issue to discuss more about fix that properly). I try to keep compatibility with previous behavior, let me fix your others suggestions and we could discuss the better after the needed done :)

Reply:
I think it's OK to resolve this.